docsite/rst/intro_inventory.rst

```diff
@@ -106,9 +106,8 @@ Variables can also be applied to an entire group at once::
 Groups of Groups, and Group Variables
 +++++++++++++++++++++++++++++++++++++
 
-It is also possible to make groups of groups and assign
-variables to groups.  These variables can be used by /usr/bin/ansible-playbook, but not
-/usr/bin/ansible::
+It is also possible to make groups of groups using the ``:children`` suffix. Just like above, you can apply variables using ``:vars``.
+These variables can be used by /usr/bin/ansible-playbook, but not /usr/bin/ansible::
```

Comment:
inventory vars get loaded no matter which executable you are using, this last line is not true